t/22-svirt.t

```diff
@@ -199,4 +208,73 @@ subtest 'SSH usage in svirt' => sub {
     is_deeply($svirt_console->get_cmd_output('echo -n "BLAFAFU"', {wantarray => 1}), ['BLAFAFU', ''], "sshVirtsh::get_cmd_output(wantarray => 1) ");
 };
 
+
+subtest open_serial_console_via_ssh => sub {
```

Comment:
actually the first argument to "subtest" can (and should) be a human readable string, e.g.

```perl
subtest 'serial_console via SSH' => sub {
```

Reply:
I changed it to `Method backend::svirt::open_serial_console_via_ssh()` as this section is only to check this specific method from backend::svirt.
wdyt?

Reply:
sure